guides/source/developers/customizations/overview.html.md

```diff
@@ -0,0 +1,24 @@
+# Overview
+
+Solidus flexibility allows changing every part of the store, both the
```

Comment:
`Solidus` -> `Solidus'` or `Solidus's`

There is not a good rule for which of the above suggestions is correct. Some people think that proper nouns (like Solidus) should be written as `Solidus'`, but it's not a strict rule.